### PR TITLE
Localize descriptions

### PR DESCRIPTION
**Pull request by noracato**
For input_elements, slides and sidebar_items!

### CODE REVIEW

**Comment by noracato** [comment]
I'll remove the move_descriptions task when the output elements are done!

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@ def info
     @title = Text.find_by_key("#{ctrl}_#{act}").try(:title) ||
       t("sidebar_items.#{s.key}.long_name") rescue nil
 
-    @description = s.description.try(:content) if s
+    @description = s.translated_description
```
Comment (sfp-kemp) [inline app/controllers/pages_controller.rb:45]: could this method become just "#description" ?

File: config/ymodel/sidebar_items.yml [hunk 2]
```diff
@@ -4,1145 +4,362 @@
...
-      of bevolkingsgroei bijvoorbeeld. Deze aannames kun je hier invullen. In de grafieken
-      zie je dan hoe alles verandert. \r\n<br/><br/>\r\nHuishoudens hebben vooral
-      behoefte aan warmte en elektriciteit. De warmtevraag neemt gestaag af door betere
-      isolatie van gebouwen. De elektriciteitsvraag is alsmaar gegroeid. Als mensen
-      rijker worden, gaan ze meer elektrische apparaten gebruiken.\r\n\r\n"
-    short_content_en: "How much energy will households need in the future?\r\nThe
-      answer depends ..."
-    short_content_nl: Hoeveel energie zullen huishoudens nodig hebben in de toekomst?
-      Dat hangt ervan af ...
+  dependent_on:
```
Comment (sfp-kemp) [inline config/ymodel/sidebar_items.yml:7]: Could you remoe the trailing whitespace?

Comment (noracato) [reply config/ymodel/sidebar_items.yml:7]: Yes! I guess I forgot that one, good catch!

**Pull request merged**